File: backend-legacy/src/main/java/com/hst/sitescrapper/model/response/UserResponse.java
package com.hst.sitescrapper.model.response;

import com.hst.sitescrapper.model.entity.UserEntity;

/**
 * @author dlgusrb0808@gmail.com
 */
public class UserResponse {
	private String id;
	private String email;
	private String name;

	public String getId() {
		return id;
	}

	public String getEmail() {
		return email;
	}

	public String getName() {
		return name;
	}

	public static UserResponse convert(UserEntity entity) {
		UserResponse response = new UserResponse();
		response.id = entity.getLoginId();
		response.email = entity.getEmail();
		response.name = entity.getName();
		return response;
	}
}

File: backend/src/main/kotlin/com/hst/scrapper/user/ui/response/UserResponse.kt
package com.hst.scrapper.user.ui.response

import com.hst.scrapper.user.domain.entity.User

/**
 * @author dlgusrb0808@gmail.com
 */
data class UserResponse(
    val id: Long,
    val loginId: String,
    val email: String,
    val name: String
) {
    companion object {
        fun fromEntity(entity: User): UserResponse {
            return UserResponse(entity.id, entity.loginId, entity.email, entity.name)
        }
    }
}